src/main/java/org/openstreetmap/atlas/checks/flag/FlaggedPoint.java

```diff
@@ -41,62 +49,54 @@ public FlaggedPoint(final LocationItem locationItem)
         this.properties = initProperties(locationItem);
     }
 
-    @SuppressWarnings("unchecked")
-    public FlaggedPoint(final Location point)
-    {
-        this.locationItem = null;
-        this.point = point;
-        this.properties = Collections.EMPTY_MAP;
-    }
-
-    @Override
-    public Iterable<Location> getGeometry()
-    {
-        return this.point;
-    }
-
-    @SuppressWarnings("unchecked")
-    @Override
-    public Map<String, String> getProperties()
-    {
-        return this.properties;
-    }
-
     @Override
     public JsonObject asGeoJsonFeature(final String flagIdentifier)
     {
         final JsonObject feature;
         final JsonObject properties;
-        if (locationItem != null)
+        if (this.locationItem != null)
         {
-            feature = locationItem.asGeoJsonGeometry();
-            properties = feature.getAsJsonObject("properties");
```

Comment:
This would return null and fail some tests after https://github.com/osmlab/atlas/pull/369